core/src/main/java/io/questdb/cairo/wal/seq/SeqTxnTracker.java

```diff
@@ -119,6 +173,40 @@ public boolean notifyOnCommit(long newSeqTxn) {
         return (stxn < 1 || writerTxn == (newSeqTxn - 1)) && suspendedState >= 0;
     }
 
+    /**
+     * Increases anti-OOM measures if possible.<br>
```

Comment:
Hard to understand pharse `anti-OOM measures`

Reply:
Improved the text in a new commit.